### DISCUSSION TITLE
'error: database "test_db" does not exist' appears when port 5432 is occupied

### DISCUSSION THREAD

**Discussion by conbainbridge** [l3atbc-datadog/pushkin#78] [General] [answered]
Encountered this issue on MacOS (commands may differ for other operating systems). At some point postgres became trapped using port 5432 for me, causing the database to fail since it needs port 5432 open.

To find out if something is clogging your port 5432:
`sudo lsof -i tcp:5432`

If it's postgres clogging the port, the following command will stop it:
`sudo pkill -u postgres`

**Comment by jkhartshorne** [comment, marked as answer]
Thanks!
